.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,85 @@
+name: Bug Report
+description: Report a bug.
+labels: ["bug"]
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: Provide a clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Provide detailed steps to replicate the bug.
+      placeholder: |
+        1. In this environment...
+        2. With this config...
+        3. Run '...'
+        4. See error...
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: What did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Current behavior
+      description: |
+        What actually happened?
+
+        Include full errors, stack traces, and/or relevant logs.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Possible reason(s)
+      description: Provide any insights into what might be causing the issue.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Suggested fix
+      description: Provide any suggestions on how to resolve the bug.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Branch, commit, and/or version
+      description: Provide the branch, commit, and/or version you're using.
+      placeholder: |
+        branch: issue-1
+        commit: abc123d
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots with descriptions to help explain your problem.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Environment details
+      description: Provide environment details (OS name and version, etc).
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional details
+      description: Provide any other additional details about the problem.
+    validations:
+      required: false
+  - type: dropdown
+    attributes:
+      label: Contribution
+      description: Can you contribute to the development of this feature?
+      options:
+        - "Yes, I can create a PR for this fix."
+        - "Yes, but I can only provide ideas and feedback."
+        - "No, I cannot contribute."
+    validations:
+      required: true
```

This is false in the template 

```suggestion
      required: false
```

---

fwiw we'll need to fix this in the template then: https://github.com/GenomicMedLab/software-templates/blob/17fd47deb4566aa4e9179fae0504225e48fe8bd2/python/%7B%7Bcookiecutter.project_slug%7D%7D/.github/ISSUE_TEMPLATE/bug-report.yaml#L85

---

would be a good use case for reusable workflows